src/hotspot/share/opto/vectornode.cpp

```diff
@@ -1815,6 +1827,17 @@ Node* VectorReinterpretNode::Identity(PhaseGVN *phase) {
   return this;
 }
 
+bool VectorReinterpretNode::implemented(int opc, uint vlen, BasicType src_type, BasicType dst_type) {
```

Comment:
`opc` is not used it. Do we need this parameter here?

Reply:
Yup not needed